src/mlpack/methods/gmm/em_fit_impl.hpp

```diff
@@ -91,31 +92,36 @@ Estimate(const arma::mat& observations,
     {
       // Store conditional probabilities into condProb vector for each
```

Comment:
Since the name changed from `condProb` to `condLogProb`, the comment should reflect that.